src/basic/string-util.c

```diff
@@ -982,6 +982,9 @@ bool string_is_safe(const char *p) {
         /* Checks if the specified string contains no quotes or control characters */
 
         for (const char *t = p; *t; t++) {
+                if ((unsigned char) *t >= 128) /* no invalid ASCII characters, i.e values between 0 and 127, inclusive. */
+                        return false;
+
```

Comment:
I don't think this is good. `string_is_safe` is used ina dozen different places, and at least e.g. in `json.c` or `kdb-util.c` we shouldn't refuse non-ASCII. (I didn't look in detail.) Part of the problem is the name of this function: "_is_safe" can mean different things to different people and in different contexts, so the name is essentially meaningless.